Makefile

```diff
@@ -54,3 +57,5 @@ clean:
 	@rm -rf .tox .eggs *.egg-info buid .pytest_cache
 	@find . -name "__pycache__" -type d -print0 | xargs -0 rm -rf
 	@find . -name "*.pyc" -delete
+
+bundle-and-validate-schema: bundle validate ## Same as successively running 'bundle' and 'validate'
```

Comment:
While I love this new option, can we have it in a different PR?

Reply:
Absolutely - will open new PR for that :+1:

Reply:
New PR for doc/Makefile suggestions: https://github.com/app-sre/qontract-schemas/pull/59